arch/arm/src/s32k3xx/s32k3xx_edma.c

```diff
@@ -154,6 +154,12 @@ struct s32k3xx_edma_s
   struct s32k3xx_dmach_s dmach[S32K3XX_EDMA_NCHANNELS];
 };
 
+#ifdef CONFIG_S32K3XX_DTCM_HEAP
+extern uint8_t DTCM_BASE_ADDR[];
+extern uint8_t DTCM_END_ADDR[];
+#define DTCM_BACKDOOR_OFFSET 0x1000000
```

Comment:
I understand that this is grouped based on Kconfig flag, but can define be moved to `Pre-processor Definitions` section?

Reply:
Ah I see that was wrong indeed, I've corrected it now.

Reply:
It is still there

Reply:
But it's in the following section now
https://github.com/apache/nuttx/blob/411ef2a5060171a6c314b2a0a041dfd863dcdb4e/arch/arm/src/s32k3xx/s32k3xx_edma.c#L79-L82
That's what you meant right?

Reply:
Oah, yes. I was just expecting `extern`s  to be separate
```
```

Reply:
Okay that was wrong indeed, I've moved the extern now as well.